tests/coalaJSONTest.py

```diff
@@ -51,6 +53,29 @@ def test_fail_acquire_settings(self):
                     found = True
             self.assertTrue(found, "Missing settings not logged")
 
+    def test_show_all_bears(self):
+        with bear_test_module():
+            retval, output = execute_coala(coala_json.main, 'coala-json', '-B')
+            self.assertEqual(retval, 0)
+            output = json.loads(output)
+            self.assertEqual(len(output["bears"]), 4)
+
+    def test_show_language_bears(self):
+        with bear_test_module():
+            retval, output = execute_coala(
+                coala_json.main, 'coala-json', '-B', '-l', 'java')
+            self.assertEqual(retval, 0)
+            output = json.loads(output)
+            self.assertEqual(len(output["bears"]), 2)
```

Comment:
@arafsheikh Could you please a test for the other attributes too? Like name, desc etc?